ci/infra/openstack/versions.tf

```diff
@@ -1,4 +1,15 @@
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.13"
```

Comment:
I doubt this is correct. We are not supporting terraform 0.13 in CaaSP, so this should always fail.

We did not follow public cloud on terraform release 0.13 because this happened once CaaSP EOF was already announced and running this upgrade was involving changes in terraform syntax config files. So in order to avoid potential regressions and having to test some config files refactor we just let it be.

Reply:
Fixed that, thank you for pointing it